.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+For high severity issues, please contact AWS Support directly.
+
+Please provide the following information:
+-->
+
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+
+### Description
+<!-- Provide detailed information about this issue -->
+
+
+
+<!--
+Please use these blocks to comment out appropriate sections
+especially for feature requests
+-->
+### Expected Behavior
+
+
+### Environment Details
+<!--
+Examples:
+* docker info
+* curl http://localhost:51678/v1/metadata
+* df -kh
+-->
+
+
+### Supporting Log Snippets
+<!--
+Please have a look at https://github.com/awslabs/ecs-logs-collector
+for the data we typically require for investigation
+-->
+
+
+### Additional Information
```

Comment:
I still find this to be implicit. At most, this could maybe be a bullet in the description section.

&lt;!-- Provide detailed information about this issue, *including any miscellaneous resources that brought you to this recommendation* --&gt;

The template is a guide, not a constraint. Folks can still add additional sections if they want to.